bin/sl-deploy.js

```diff
@@ -5,18 +5,21 @@
 var Parser = require('posix-getopt').BasicParser;
 var debug = require('debug')('strong-deploy');
 var defaults = require('strong-url-defaults');
-var getPackageInfo = require('../lib/package').getPackageInfo;
 var defaultPackagePath = require('../lib/package').getPackagePath;
 var deploy = require('../');
 var fs = require('fs');
+var getPackageInfo = require('../lib/package').getPackageInfo;
+var glb = require('strong-globalize');
 var maybeTunnel = require('strong-tunnel');
 var path = require('path');
 
-function printHelp($0, prn) {
-  var USAGE = fs.readFileSync(require.resolve('./sl-deploy.txt'), 'utf-8')
-      .replace(/%MAIN%/g, $0)
-      .trim();
+glb.setRootDir(path.resolve(__dirname, '..'));
+glb.setDefaultLanguage();
 
+function printHelp($0, prn) {
+  var USAGE = glb.t('sl-deploy.txt')
```

Comment:
this surprises me, I'd have expecte `glb .f('sl-deploy.txt')`. How do you distinguish between a msg ID `sl-deploy.txt`, and a file? Or, are you not distinguishing, so it becomes an implementation detail, the string can be in the JSON, or an entire text file, but it doesn't matter here, at the point of use? Thinking about that... I  think I like it!